### ISSUE TITLE
#5 Use semantic versioning and keep data uniquely identifiable 

### ISSUE THREAD

**Issue by emhart** [emhart/10-simple-rules-data-storage#35]
Data sets need to versioned and identifiable.  Ideally your data will follow a semantic versioning if it's periodically revised.  Each version should have a way to be uniquely identified.  This could be a DOI, PURL, ARK, etc...  Streaming data should also be addressed here.  E.g. if you have data streaming in, what is versioned, what is identified?

Refs:  #9 and #27

**Comment by PBarmby** [comment]
Do we need links/references for the items in the first paragraph?
Did the "ambiguous example above" get lost?

**Comment by snim2** [comment]
Yes, sorry the example got changed after feedback and I missed that. I've opened PR #55 to resolve that.

There are links to the items in the first paragraph (at least in the Markdown), unless I have misunderstood you. I wasn't sure whether websites like this should be formatted as hyperlinks, footnotes or citations, but I thought that might be something to look at over the whole document, to make sure we are all consistent.
